tests/tests/test_fault_tolerance.py

```diff
@@ -93,7 +127,7 @@ def test_deployed_during_network_outage(self, standard_setup_one_client_bootstra
 
         mender_device = standard_setup_one_client_bootstrapped.device
 
-        Helpers.gateway_connectivity(mender_device, False)
+        self.manipulate_network_connectivity(mender_device, False)
```

Comment:
Suggestion:
Since this is a static method, is not cleaner to call it with the class-name, instead of the object reference?

Reply:
Okey, it is the same to me, I will change it.

When the static method is from the same class, I usually use `self.`